var/README.namelist

```diff
@@ -470,8 +470,11 @@ Description of WRFDA namelist variables, defined in Registry/registry.var
                                                  ; first assimilation cycle. If there are not enough data 
                                                  ; (according to "VARBC_NOBSMIN") on the first cycle, the 
                                                  ; next cycle will perform a coldstart again.
-  use_clddet_mmr                      = false    ; .true. :use the MMR scheme to conduct cloud detection for infrared radiance
-  use_clddet_ecmwf                    = false    ; .true. :use the ECMWF operational scheme to conduct cloud detection for infrared radiance.
+  use_clddet                      = 0            ; 0: off
```

Comment:
Default setting is 2 from registry.var.

Reply:
Ahh yes, i forgot to change the default value when i added the default in brackets, sorry. The change has now been made.